README-multiple-instance.md

```diff
@@ -0,0 +1,72 @@
+ - run_multiple_instance.sh  - Runs multiple instances of the server client and mongodb on the same machine or across different machines.
+   	 -2 run modes - bare metal run (default run mode 0) and container run (run mode 1)
+
+ - multiple_instance_config.json - The input configuration file for multiple instances. Each instance requires these parameters:
+ 	  {
+  		"server_ip":"127.0.0.1", //IP address where the server runs
+  		"server_port":"9001",  // Port the server listens to
+  		"db_name":"node-els-db1", // Database name
+  		"db_ip":"127.0.0.1", // Database IP address
+  		"db_port":"27018" // Port database listens to
+ 	  }
+ - multiple_instance_post_process.py - Script to post process and summarize data from all the instances
+    -  Post processes data from each instance to generate a summary data file.
+    -  Generates live throughput and response time graphs.
+
+ - a results_multiple_instance will be created after the run which contains all the result directories for which are designated by the date and time stamp.
+ 	- log directory with individual instance logs and master log designated by the date and time stamp.
+ 	- Master RTdata file - summarizes data (min,mean max, 95 and 99 percentile response time and Throughput) from all the instances for every given interval.
+ 	- Summary Throughput and response time graphs.
+ 	- Master summary file - Provides the combined throughput of all the instances, 99 percentile response time, the number of instances, concurrency used, a detailed summary of response times and each instance summary.
+
+In addition there are 3 supporting scripts:
+	- start-server.sh - Used to start server and mongodb during a bare metal run.
+		- requires NODE_PATH (path to node binary) to be set before the run.
+	- stop-server.sh - Used to stop server and mongodb during a bare metal run
+	-container-startup.sh - Used to start containers.
+
+Steps in a typical run:
+	- The multiple_instance_config.json file is processed to create server and client config files for each instance.
+	- The server code is copied to a remote server (IP provided in the config file) and each instance server is started.
+	- A curl command is issued to check if every server instance is up and running.
+	- Each instance client is started once all the servers are up.
+	- Each client waits till all the instances have finished loading database.
+	- The post processing script starts in the background once all the clients have their first summary data available.
+	- The server and mongodb stops once post processing is completed.
+
+Setup instructions:
+Auto-login setup between the client and the server
+    - SSH auto login needs to be set up between the client and the server.RSH and RCP uses this auto login to remote into the server and start the server.
+      -Steps for auto login setup
+		-$ ssh-keygen -t rsa -b 2048
```

Comment:
looks like the markup is still kind of broken. It might be related to spaces vs tab and indentation levels